comments/screenshot.py

```diff
@@ -519,15 +543,12 @@ def print_comments_availability_on_page(comments, detailed=False):
                                     time.sleep(3)
                                     if is_new_layout(page):
                                         # Bypass "See this post in..."
-                                        see_this_post_in_button = page.locator('#bottom-sheet button.continue').first
-                                        if see_this_post_in_button.is_visible():
-                                            print("See this post in... [CONTINUE]")
-                                            see_this_post_in_button.dispatch_event('click')
+                                        bypass_see_this_post_in(page)
 
                                         comment_loc = page.locator(selector).first
                                         if comment_loc:
                                             # If replies are expanded toggle them
-                                            expanded_loc = comment_loc.locator('#comment-fold-button[aria-expanded="true"]').first
+                                            expanded_loc = comment_loc.locator('#utton[aria-expanded="true"]').first
```

Comment:
is this a typo? should it be `#utton` or `#button` ?

Reply:
```suggestion
                                            expanded_loc = comment_loc.locator('#button[aria-expanded="true"]').first
```

Reply:
Typo 😝 
Should have been `button`, fixed in the last commit.